tests/models/test_siglip.py

```diff
@@ -0,0 +1,61 @@
+import pytest
+from PIL import Image
+
+from vision_agent_tools.models.siglip import Siglip
+
+
+def test_zero_shot_image_classification(model):
+    image = Image.open("tests/shared_data/images/car.jpg")
+    labels = ["2 cars", "1 car", "1 airplane", "1 boat"]
+
+    result = model(
+        image=image,
+        labels=labels,
+    )
+
+    assert result is not None
+    assert len(result["labels"]) == 4
+    assert len(result["scores"]) == 4
+    assert all(isinstance(label, str) for label in result["labels"])
+    assert all(isinstance(score, float) for score in result["scores"])
```

Comment:
should we maybe also test that all scores sum up to 1?

Reply:
oh, actually I don't think this is true, right? because it uses `sigmoid` instead of softmax in the implementation.

Reply:
Yes! It uses Sigmoid